bats/update-tables.bats

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env bats
+
+setup() {
+    load $BATS_TEST_DIRNAME/helper/common.bash
+    export PATH=$PATH:~/go/bin
+    export NOMS_VERSION_NEXT=1
+    cd $BATS_TMPDIR
+    mkdir "dolt-repo-$$"
+    cd "dolt-repo-$$"
+    dolt init
+}
+
+teardown() {
+    rm -rf "$BATS_TMPDIR/dolt-repo-$$"
+}
+
+@test "update table using csv" {
+    run dolt table create -s `batshelper 1pk5col-ints.schema` test
+    [ "$status" -eq 0 ]
+    run dolt table import -u test `batshelper 1pk5col-ints.csv`
+    [ "$status" -eq 0 ]
+    [[ "$output" =~ "Rows Processed: 2, Additions: 2, Modifications: 0, Had No Effect: 0" ]] || false
+    [[ "$output" =~ "Import completed successfully." ]] || false
+}
+
+@test "update table using schema with csv" {
+    run dolt table create -s `batshelper 1pk5col-ints.schema` test
+    [ "$status" -eq 0 ]
+    run dolt table import -u -s `batshelper 1pk5col-ints.schema` test `batshelper 1pk5col-ints.csv`
+    [ "$status" -eq 1 ]
+    [[ "$output" =~ "schema is not supported for update operations" ]] || false
+}
+
+@test "update table using csv with newlines" {

```

Comment:
@bheni I added a test here for importing CSVs with new lines too.

Reply:
I don't think you want to conflate this with what you are addressing in this PR.

Reply:
Okay, I can make this a separate one